python/tests/test_extractor.py

```diff
@@ -0,0 +1,31 @@
+from io import StringIO
+import pytest
+from energuide import extractor
+
+
+@pytest.fixture
+def csv_string():
+    return '''EVAL_ID,EVAL_TYPE,HOUSEREGION,WEATHERLOC,ENTRYBY,CLIENTCITY,CLIENTPCODE,CLIENTNAME,TELEPHONE,\
```

Comment:
If the input to the whole thing is a filepath, let's make the fixture a filepath. You can use `tmpdir` to give you a temporary filesystem per-test for writing the fixture data to.